src/controller/user/controller.go

```diff
@@ -167,6 +172,16 @@ func (c *controller) Count(ctx context.Context, query *q.Query) (int64, error) {
 }
 
 func (c *controller) Delete(ctx context.Context, id int) error {
+	// cleanup project member with the user
+	if err := c.memberMgr.DeleteMemberByUserID(ctx, id); err != nil {
+		return err
```

Comment:
The error can be confusing b/c it doesn't contain information to explicitly let user know it happens when deleting the membership.

Reply:
Updated

Reply:
updated